### ISSUE TITLE
Menu tweak URL can't be edited after creation

### ISSUE THREAD

**Issue by PierreMesure** [Platoniq/decidim-module-decidim_awesome#155]
Hi,

I found a weird behaviour in Menu Tweaks, seems unlikely that it's by design.

After adding a menu tweak, it's impossible to edit the URL.

<img width="1068" alt="image" src="https://user-images.githubusercontent.com/7223028/153003486-21685d3e-93d6-4c3d-b63e-9ff4efbae2bb.png">

Of course, it's possible to remove it and add a new one. But not so convenient.

**Comment by PierreMesure** [comment]
I had a closer look at the code and my understanding is that it's [this line](https://github.com/Platoniq/decidim-module-decidim_awesome/blob/0807c69f9508a703052059840fe54883ce840457/app/views/decidim/decidim_awesome/admin/menu_hacks/_form.html.erb#L3) that causes it.

I don't really see how it could have been set this way by mistake so there must be some intent behind it. @microstudi, what was your thinking for this behaviour?

**Comment by microstudi** [comment]
Yes, when making this I found some technical challenges that make me took this decision. There's probably a better way to do it, but given the (very low) cost of removing and creating a new menu instead of editing it, I probably won't tackle this issue anytime soon. Happy to accept contributions though.
